addons/miscFixes/patchCUP/config.cpp

```diff
@@ -43,18 +54,35 @@ class CfgVehicles {
         };
         class UserActions {}; // clear all user actions (not a big deal)
     };
-    // Fix the M1038 back seat
-    class Car_F;
+    // Fix the M1038 back seat and attenuation
     class CUP_nHMMWV_Base: Car_F {
         class CargoTurret;
+        class Turrets: Turrets {
+            class MainTurret;
+        };
     };
     class CUP_nM1038_Base: CUP_nHMMWV_Base {
-        class Turrets {
+        class Turrets: Turrets {
```

Comment:
`Updating base class ->Turrets, by z\potato\addons\miscFixes\patchcup\config.cpp/CfgVehicles/CUP_nM1038_Base/Turrets/ (original CUP\WheeledVehicles\CUP_WheeledVehicles_NewM998\config.bin)`


```suggestion
        class Turrets {
```

Reply:
Ty, missed that one